pkgs/development/compilers/yasm/default.nix

```diff
@@ -15,4 +15,6 @@ stdenv.mkDerivation rec {
     license = licenses.bsd2;
     platforms = platforms.unix;
   };
+
+  depsBuildBuild = [ buildPackages.stdenv.cc ];
```

Comment:
meta should be last and this must also target staging.

Reply:
done